src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -34,7 +34,7 @@ public sealed class ZipkinExporterOptions
         /// Gets or sets Zipkin endpoint address. See https://zipkin.io/zipkin-api/#/default/post_spans.
```

Comment:
Shall we update the comment and/or the README.md with the environment variable information? (maybe also include the specs link: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md for whoever reads code/uses this feature in future)

Reply:
@xiang17 - I updated the readme doc.